examples/terraform/advanced/main.tf

```diff
@@ -0,0 +1,308 @@
+module "bootstrap_token" {
```

Comment:
using this still doesn't work, not sure why, k8s doesn;t come up, the static pods

Reply:
there can be too many things... is it just the token? 

I would argue supporting any ways to generate secrets outside of `machinery` package is a challenge on its own

Reply:
initially it was the tokens, but it's just seems like kubelet auth errors or something of that sort. I'll get logs and check later

Reply:
the issue was with the bootstrap token , it had a newline prefix. The example now works.